src/main/scala/com/timeout/healthchecks/HealthChecks.scala

```diff
@@ -0,0 +1,20 @@
+package com.timeout.healthchecks
+
+import java.net.{Socket, URL}
+
+import cats.syntax.all._
```

Comment:
Could we use more targeted imports? It's a good habit going forward so we don't massively increase compile time